request.go

```diff
@@ -244,6 +244,19 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				continue
 			}
 
+			if v.Kind() == reflect.String {
+				stringValue := v.Interface().(string)
+
+				// The field may or may not be a pointer to a string; handle both cases
+				if fieldValue.Kind() == reflect.Ptr {
+					fieldValue.Set(reflect.ValueOf(&stringValue))
```

Comment:
@shwoodard I believe that part of the point with this changeset is that, in the case where `fieldValue` is a `*string`, `nil` would represent a value that should not be set, but `""` should be set. This allows for a user to update a field to an empty string when they want to. Instead of the empty string being interpreted as a missing value.

Thoughts?